### PR TITLE
feat: or-507 add managed by field in search api

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: src/OrganisationRegistry.Api/Configuration/OrganisationRegistryConfiguration.cs [hunk 1]
```diff
@@ -25,5 +18,23 @@ public class OrganisationRegistryConfiguration : IOrganisationRegistryConfigurat
             Kbo = new KboConfiguration(configuration, terminationConfiguration);
             Authorization = new AuthorizationConfiguration(authorizationConfiguration);
         }
+
```
Comment (koenmetsu) [inline src/OrganisationRegistry.Api/Configuration/OrganisationRegistryConfiguration.cs:21]: autoformatted, not sure if I 100% like it, but it's consistent.

**Comment by koenmetsu** [comment]
One of us is going to have to merge... ;)

File: src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs [hunk 2]
```diff
@@ -38,24 +39,99 @@ public class Organisation :
...
+            IOptions<ElasticSearchConfiguration> elasticSearchOptions,
+            IOrganisationManagementConfiguration organisationManagementConfiguration) : base(logger)
         {
             _elastic = elastic;
             _elasticSearchOptions = elasticSearchOptions.Value;
+            _organisationManagementConfiguration = organisationManagementConfiguration;
         }
 
-        public async Task<IElasticChange> Handle(DbConnection dbConnection, DbTransaction dbTransaction, IEnvelope<OrganisationCreated> message)
+        public async Task<IElasticChange> Handle(
```
Comment (W0dan) [inline src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs:61]: hmmm is this really domain logic ? somehow feels out of place in the AR. This is code I would place in the application layer I think.

File: src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs [hunk 3]
```diff
@@ -1,17 +1,38 @@
...
+            Capacities = new List<OrganisationCapacity>();
+            Parents = new List<OrganisationParent>();
+            FormalFrameworks = new List<OrganisationFormalFramework>();
+            Buildings = new List<OrganisationBuilding>();
+            Locations = new List<OrganisationLocation>();
+            Bodies = new List<OrganisationBody>();
+            BankAccounts = new List<OrganisationBankAccount>();
+            OpeningHours = new List<OrganisationOpeningHour>();
+            Regulations = new List<OrganisationRegulation>();
+            Validity = new Period(); // default: infinitely valid
```
Comment (W0dan) [inline src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs:33]: maybe change to 'Period.Infinity' ? -> then the comment can go

File: src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs [hunk 4]
```diff
@@ -2,20 +2,21 @@ namespace OrganisationRegistry.ElasticSearch.Projections.Organisations
 {
```
Comment (W0dan) [inline src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs:2]: this is quite a PITA: 
image

File: src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs (hunk 3 above)
Comment (koenmetsu) [reply src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.cs:33]: I think the comment can go even without a code change, but I don't mind making it more explicit :+1:

**Comment by koenmetsu** [comment]
@W0dan done.

File: src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs (hunk 4 above)
Comment (W0dan) [reply src/OrganisationRegistry.ElasticSearch.Projections/Organisations/Organisation.cs:2]: I'll look into it

File: src/OrganisationRegistry.ElasticSearch.Projections/Program.cs [hunk 5]
```diff
@@ -118,6 +120,14 @@ public static async Task Main(string[] args)
                         .Configure<OpenIdConnectConfigurationSection>(
                             hostContext.Configuration.GetSection(OpenIdConnectConfigurationSection.Name));
 
+                    builder
+                        .AddSingleton<IOrganisationManagementConfiguration>(
+                            _ =>
+                                new OrganisationManagementConfigurationConfiguration(
```
Comment (W0dan) [inline src/OrganisationRegistry.ElasticSearch.Projections/Program.cs:126]: quite redundant naming

File: src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.OrganisationFormalFramework.cs [hunk 6]
```diff
@@ -19,7 +19,7 @@ protected OrganisationFormalFramework()
             {
                 FormalFrameworkName = string.Empty;
                 ParentOrganisationName = string.Empty;
-                Validity = new Period(); // default infinitely valid
+                Validity = Period.Infinite(); // default infinitely valid
```
Comment (W0dan) [inline src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.OrganisationFormalFramework.cs:22]: comment may be removed

**Review by W0dan — APPROVED** [review]

**Pull request merged**